.github/workflows/e2eaiok_nightly_pypi.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build wheel
       run: |
         pip install build wheel
-        release_version=$(cat version | head -1)
+        release_version=$(cat e2eAIOK/version | head -1)
```

Comment:
why we are putting version in e2eAIOK? version is for all components

Reply:
The original location of `version` file can not properly be ready when you run `python setup.py install`

Reply:
`version` is for all components in e2eAIOK

Reply:
I see, I'll merge this PR firstly and test with action then.